core/src/mindustry/editor/WaveGraph.java

```diff
@@ -171,6 +171,18 @@ public void rebuild(){
 
         colors.clear();
         colors.left();
+        colors.button("@waves.units.hide", Styles.fullTogglet, () -> {
+            if (hidden.size == usedCopy.size){
```

Comment:
This formatting is *completely* off, consult the style guidelines.

Reply:
I could have sworn i pushed a format fix to all three prs...
Welp fixing this now i guess.